bin/search.py

```diff
@@ -26,6 +26,7 @@
 
 from lib.DatabaseLayer import cvesForCPE, getCVEs, getFreeText, getCVEIDs, searchCVE
 from lib.CVEs import CveHandler
+from CveXplore.common.cpe_converters import split_cpe_name
```

Comment:
Here, we need to remember that this function has been added in https://github.com/cve-search/CveXplore/pull/269 which is after CveXplore's last release [v0.3.24](https://github.com/cve-search/CveXplore/releases/tag/v0.3.24); CVE-Search only has CveXplore as a pip dependency (`cvexplore==0.3.24`). However, this pull requests adds the same function in `lib/cpe_conversion.py` and uses it elsewhere, so it might be best to use it here, too, rather than importing it from the external library.

```suggestion
rom lib.cpe_conversion import split_cpe_name
```

Reply:
That's right, this is a mistake, I meant to use lib.cpe_conversion everywhere